src/test/java/io/swagger/oas/test/examples/ExampleBuilderTest.java

```diff
@@ -944,4 +945,57 @@ public void testSwos126() throws Exception {
         assertTrue(jsonExample.contains("\"date\" : \"2019-08-05\""));
         assertTrue(jsonExample.contains("\"dateTime\" : \"2019-08-05T12:34:56Z\""));
     }
+
+    @Test
+    public void testNullExampleSupportOAS3(){
+
+        ParseOptions options = new ParseOptions();
+        options.setResolve(true);
+        options.setResolveFully(true);
+
+        OpenAPI openAPI = new OpenAPIV3Parser().read("src/test/swagger/null-examples-oas3.yaml", null, options);
+
+        ApiResponse response = openAPI.getPaths().get("/object-with-null-example").getGet().getResponses().get("200");
+        Example example = ExampleBuilder.fromSchema(response.getContent().get("application/json").getSchema(), null, ExampleBuilder.RequestType.READ);
+        String output = Json.pretty(example);
+        assertNull(output);
```

Comment:
Hi, the expected result for the example here should be `foo: null` instead of the example being null

Reply:
@gracekarina no. The object itself has `null` as the example value:
```yaml
    ObjectWithNullExample:
      type: object
      properties:
        foo:
          type: string
      nullable: true
      example: null   # <----
```

Reply:
the ExampleBuilder will build an example, of the `foo` property and it may be null,
that's why `foo: null`;

Reply:
@hkosova the parser will return this: 
```
    ObjectWithNullExample:
      type: object
      properties:
        foo:
          type: string
      nullable: true
      example: null   # <----
```
But  inflector's Example Builder will try to recreate an example of the property according to the type.